README.md

```diff
@@ -57,7 +57,23 @@ self.addEventListener('periodicsync', event => {
 
 # Security and Privacy
 
-TODO
+One-shot Background Sync allows the web page’s logic to live a little longer (service worker execution time) after the page has been closed by the user. Periodic Background Sync extends it to potentially infinite time thereafter. Here are some privacy concerns:
+
+* There is no notification to the user.
```

Comment:
Why can't implementations do something like push notifications, and force a user visible result to be shown? (E.g. a notification, or a tab title update, or ...)

Reply:
I've updated to - " There might be no notification to the user" since this is browser-implementation dependent.